drivers/clock_control/Kconfig.nrf

```
@@ -32,32 +32,38 @@ choice CLOCK_CONTROL_NRF_SOURCE
 
 config CLOCK_CONTROL_NRF_K32SRC_RC
 	bool "RC Oscillator"
+	select NRFX_CLOCK_LF_SRC_RC
```

Since the added options depend on `NRFX_CLOCK` and that option is selected `if !CLOCK_CONTROL_NRF_FORCE_ALT`, I think such condition needs to be added also for their selection, like it is done for `NRFX_CLOCK_LFXO_TWO_STAGE_ENABLED`.
```suggestion
	select NRFX_CLOCK_LF_SRC_RC if !CLOCK_CONTROL_NRF_FORCE_ALT
```

---

This should be done in all the added `select` lines.

---

right, done.